src/components/feature-details.vue

```diff
@@ -74,29 +100,37 @@ export default {
   },
 
   data: () => ({
-    featurePlotLink: null,
     featureDetails: [],
-    hasData: false
+    hasLoaded: false
   }),
 
+  computed: {
+    htmlPlots() {
+      return this.featureDetails.filter(({ url }) => hasExtension('html')(url));
+      // For testing :: return [ { url: 'FAKE_DATA/plot_1575300895.767069.html' } ];
```

Comment:
remove?

Reply:
Unfortunately I can't access the actual urls because I can't access the DELTARES network where the server is running. So I need these for a while yet - will delete when appropriate.